src/components/ens-registration/ConfirmContent/RegisterContent.tsx

```diff
@@ -33,9 +41,14 @@ const RegisterContent = ({
       </Box>
       <Box height="1/5">
         <Inline alignHorizontal="justify" alignVertical="center">
-          <Text color="secondary80" size="16px" weight="bold">
-            {lang.t('profiles.confirm.set_ens_name')} 􀅵
-          </Text>
+          <ButtonPressAnimation
+            onPress={openPrimaryENSNameHelper}
+            scaleTo={0.9}
+          >
+            <Text color="secondary80" size="16px" weight="bold">
+              {lang.t('profiles.confirm.set_ens_name')} 􀅵
```

Comment:
looking good, can we make that the only text where you can press is the icon? the rest of the text shouldn't be pressable, that's how the rest of the explainer sheets work in the app